repository/src/main/resources/xsd/repositorytypes.xsd

```diff
@@ -1028,17 +1023,13 @@
 			<xs:enumeration value="Tenor"/>
 		</xs:restriction>
 	</xs:simpleType>
-	<xs:simpleType name="Usage_t">
-		<xs:restriction base="xs:string"/>
-	</xs:simpleType>
 	<xs:simpleType name="Version_t">
-		<xs:restriction base="xs:string">
+		<xs:restriction base="xs:token">
 			<xs:annotation>
-				<xs:documentation>Enumerated FIX versions or major.minor or date as
-					yyyymmdd of any protocol
+				<xs:documentation>Uniquely identifies a revision. Many formats are impossible, including but not limited to protocol version label, semantic versioning, date, or simple integer.
```

Comment:
Should it rather be "possible" and not "impossible"?

Reply:
Yep, thanks